packaging/hudi-presto-bundle/pom.xml

```diff
@@ -76,7 +76,12 @@
                   <include>org.apache.hbase:hbase-common</include>
                   <include>org.apache.hbase:hbase-protocol</include>
                   <include>org.apache.hbase:hbase-server</include>
+                  <include>org.apache.hbase:hbase-annotations</include>
                   <include>org.apache.htrace:htrace-core</include>
+                  <include>com.yammer.metrics:metrics-core</include>
+                  <include>com.google.guava:guava</include>
+                  <include>commons-lang:commons-lang</include>
```

Comment:
Can you also shade commons-lang and protobuf ?

Reply:
I think shading protobuf causes other issues. Also these dependencies were not part of presto runtime that is why I did not feel the need to shade them.

Reply:
Let me quickly try shading them again, and remember what issues I was running into.

Reply:
I am able to shade as well as relocate `commons-lang` and `protobuf`